src/particles.cpp

```diff
@@ -190,8 +190,9 @@ void ServerParticleTexture::serialize(std::ostream &os, u16 protocol_ver,
 	FlagT flags = 0;
 	if (animated)
 		flags |= FlagT(ParticleTextureFlags::animated);
-	if (blendmode != BlendMode::alpha)
-		flags |= FlagT(blendmode) << 1;
+	// Default to `blend = "alpha"` for older clients that don't support `blend = "clip"`
+	flags |= FlagT(protocol_ver < 47 && blendmode == BlendMode::clip
+			? BlendMode::alpha : blendmode) << 1;
```

Comment:
How about using some indent levels for a (subjectively) friendlier appearance?
```suggestion
	flags |= FlagT(
		(protocol_ver < 47 && blendmode == BlendMode::clip)
			? BlendMode::alpha
			: blendmode
	) << 1;
```

Reply:
or:
```cpp
auto sent_blendmode = (protocol_ver < 47 && blendmode == BlendMode::clip) ? BlendMode::alpha : blendmode;
flags |= FlagT(sent_blendmode) << 1;
```